pkg/scheduler/apis/config/v1alpha1/defaults.go

```diff
@@ -78,11 +78,11 @@ func SetDefaults_KubeSchedulerConfiguration(obj *kubescedulerconfigv1alpha1.Kube
 		obj.MetricsBindAddress = net.JoinHostPort("0.0.0.0", strconv.Itoa(ports.InsecureSchedulerPort))
 	}
 
-	if len(obj.LeaderElection.LockObjectNamespace) == 0 {
-		obj.LeaderElection.LockObjectNamespace = kubescedulerconfigv1alpha1.SchedulerDefaultLockObjectNamespace
```

Comment:
keep defaulting setting the LockObjectNamespace field. we want the files we generate to work for n-1 schedulers that only read that field

Reply:
sorry, I meant *instead of* the new field. Defaulting both means `kube-scheduler --write-config-to=config.yaml` would write a file with both fields populated, which is pretty confusing. Same comment for LockObjectName

Reply:
Fixed.